commit d74d866be83c112b7d201ca5467ea68111eedbf6
Author: Hubert Figuière <hub@figuiere.net>
Date:   Sun Jan 11 11:18:21 2026 -0500

    appstream: Add vcs-browser and caption
    
    These are warning on flathub
    
    Signed-off-by: Hubert Figuière <hub@figuiere.net>

diff --git a/desktop/org.librecad.librecad.appdata.xml b/desktop/org.librecad.librecad.appdata.xml
index d1dcb7d1e..05ac0ead8 100644
--- a/desktop/org.librecad.librecad.appdata.xml
+++ b/desktop/org.librecad.librecad.appdata.xml
@@ -11,9 +11,11 @@
     </description>
     <launchable type="desktop-id">librecad.desktop</launchable>
     <url type="homepage">https://librecad.org/</url>
+    <url type="vcs-browser">https://github.com/LibreCAD/LibreCAD</url>
     <screenshots>
         <screenshot type="default">
             <image>https://dokuwiki.librecad.org/lib/exe/fetch.php/librecad_screeshot_2.2.png</image>
+            <caption>Main Window</caption>
         </screenshot>
     </screenshots>
     <provides>
